docs/developers/set-metrics.md

```diff
@@ -1,3 +1,63 @@
 # Monitoring Application
 
-> TODO
\ No newline at end of file
+If your application has exposed metrics, you can easily setup monitoring system
+with the help of `metric` capability.
+
+Let's run [`christianhxc/gorandom:1.0`](https://github.com/christianhxc/prometheus-tutorial) as an example app.
+The app will emit random latencies as metrics.
+
+```console
+$ vela svc deploy metricapp -t webservice --image christianhxc/gorandom:1.0 --port 8080
+```
+
+Then add metric by:
+
+```console
+$ vela metric metricapp
+Adding metric for app metricapp
+⠋ Deploying ...
+✅ Application Deployed Successfully!
+Showing status of service(type: webservice) metricapp deployed in Environment myenv
+Service metricapp Status:	 HEALTHY Ready: 1/1
+	metric: scheme=http
+        enabled=true
+        format=prometheus
+        path=/metrics
+
+Last Deployment:
+	Created at: 2020-10-31 16:37:11 +0800 CST
+	Updated at: 2020-10-31T16:59:19+08:00
+```
+
+The metrics trait will automatically discover port and label to monitor if no parameters specified.
+If more than one ports found, it will choose the first one by default.
+
+
+## Verify that the metrics are collected on prometheus
```

Comment:
I tried query `avg(container_cpu_usage_seconds_total{container!="POD",namespace="default",pod=~"metricapp.*"}) * 100`, but  it seems the metrics weren't collected. Or did I miss any prerequisites?

Reply:
this is related with #491, we should discuss how to fix